Application/EdFi.Ods.Api/Caching/DescriptorsCache.cs

```diff
@@ -55,7 +57,7 @@ private bool HasLastDatabaseSynchronizationExpired
                 }
 
                 var totalSeconds = (currentTime - lastSynced).TotalSeconds;
-                return totalSeconds > DatabaseSynchronizationExpirationSeconds;
+                return totalSeconds > _absoluteExpirationPeriod.TotalSeconds;
```

Comment:
This property is probably not well named, its really used more like a quiet period setting.  Requests for descriptors from the cache will refresh the descriptors if there's a miss that gets resolved, and will then attempt to refresh the entire cache under the assumption that the cache is out of sync.  If this quiet period is still in effect, it will not do that entire cache refresh.  This doesn't actually cause the cache to get cleared out, we'd have to update the places where we're inserting to the cache as well as updating the underlying cache provider to respect these parameters.

Reply:
So, this is just primarily to stop bad client data from forcing full descriptor cache refreshes with every miss. The full refresh will only happen once a minute, at most (as originally written).

Reply:
Yep, exactly.

Reply:
Reverted all changes to DescriptorsCache -- and implemented `ExpiringConcurrentDictionaryCacheProvider`.